index.html

```diff
@@ -30,279 +33,326 @@
-        feature or website. Look through some of my work and experience! If
-        you like what you see and have a project you need coded, don’t
-        hesitate to contact me.
+    <section class="main-section">
+      <h1 class="primary-text">I’m Ankit kush. Glad to see you!</h1>
+      <p class="paragraph">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact me.
       </p>
-      <a href="#" class="connect about-connect">Let's connect</a>
-      <ul class="social about-social">
+      <h3>
+        <a class="connect" href="https://www.linkedin.com/in/ankit26k/">
+          LET'S CONNECT</a
+        >
+      </h3>
+
+      <ul class="social">
         <li>
           <a href="https://twitter.com/ankit26k">
-            <img class="icon" src="images/twitter.png" alt="twitter" /></a>
+            <img class="icon" src="images/twitter.png" alt="twitter"
+          /></a>
         </li>
 
         <li>
           <a href="https://www.linkedin.com/in/ankit26k/">
-            <img class="icon" src="images/Linkedin.png" alt="linkedin" /></a>
+            <img class="icon" src="images/Linkedin.png" alt="linkedin"
+          /></a>
         </li>
 
         <li>
           <a href="https://twitter.com/ankit26k">
-            <img class="icon meta" src="images/Vector1.png" alt="meta" /></a>
+            <img class="icon meta" src="images/Vector1.png" alt="meta"
+          /></a>
         </li>
 
         <li>
           <a href="https://github.com/ankitt26">
-            <img class="icon" src="./images/vector2.png" alt="github" /></a>
+            <img class="icon" src="images/vector2.png" alt="github"
+          /></a>
         </li>
 
         <li>
           <a href="https://www.instagram.com/ankit16t/">
-            <img class="icon" src="./images/vector3.png" alt="twofin" /></a>
+            <img class="icon" src="images/vector3.png" alt="twofin"
+          /></a>
         </li>
       </ul>
-      <button class="about-btn btn_btn">Get my resume</button>
     </section>
 
-    <!-- section 2 -->
+    <!-- Work section !-->
+    <section class="works" id="portfolio">
+      <div class="cards" id="one1">
+        <div class="block1">
+          <img
+            class="card-img"
+            src="images/Snapshoot.png"
+            alt="tonic project"
+          />
+        </div>
+        <div class="block2">
+          <div class="box1">
+            <h2 class="title1">Tonic</h2>
+            <ul class="frame1">
+              <li class="A Li">CANOPY</li>
+              <li class="BC Li">
+                <i class="bi bi-circle-fill icon"></i> Back End dev
+              </li>
+              <li class="BC Li"><i class="bi bi-circle-fill icon"></i> 2019</li>
+            </ul>
+          </div>
```

Comment:
- [ ] Your projects' popups have one problem in all screen sizes. they don't follow the figma design properly. as on the top they are okay but in the bottom you dont see the border of the popup.So you dont see where does they end.
project requirement : You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
kindly see the difference between your popups and the figma design and consider fixing them:
your design
![re](https://user-images.githubusercontent.com/96683843/231834879-a6f3f803-6f85-4e1e-8dc3-c5b379dc4a98.jpg)
![fd](https://user-images.githubusercontent.com/96683843/231834897-9d567bcc-51bf-48ed-93af-e80b9d9a87e3.jpg)


figma design
![as](https://user-images.githubusercontent.com/96683843/231834818-2889eec1-6863-4352-a608-f6eaf00743fe.jpg)
![qw](https://user-images.githubusercontent.com/96683843/231834845-ed41cc50-a3da-4be3-9f44-2eec2e550b7c.jpg)